stories/NewSelectField/newSelectField.md

```diff
@@ -0,0 +1,33 @@
+
+# select field component
+
+&nbsp;
+
+### Example of usage in bourbon
+```
+{{new-bourbon-select-field content=petsArray prompt="Select an animal..." value="Select an animal..."}}
+```
+&nbsp;
+
+## Properties
+| propertyOptions | type | defaultValue | isRequired | description | options |
+|----------|:----------|:--------------|:------------|:-------------|:------|
+| classNames | string | NewBourbonSelectField | false | can modify styles and spacing for the select field ||
+| prompt | string | null | false | if you want a placehoder prompt to guide user| for example, "Select a Salesforce object..."|
+| content | array | null | true | list of content to be displayed in select field | contents of array can be Ember.Object, JS Object, Ember model|
```

Comment:
the content can also be strings and numbers, right?

Reply:
yep.  :)

Reply:
sorry but... strings too right? 😂